llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp

```diff
@@ -2168,15 +2170,35 @@ bool DWARFVerifier::verifyDebugStrOffsets(
 
 void OutputCategoryAggregator::Report(
     StringRef s, std::function<void(void)> detailCallback) {
-  Aggregation[std::string(s)]++;
+  this->Report(s, "", detailCallback);
+}
+
+void OutputCategoryAggregator::Report(
+    StringRef category, StringRef sub_category,
+    std::function<void(void)> detailCallback) {
+  std::string category_str = std::string(category);
+  AggregationData *Agg = &Aggregation[category_str];
```

Comment:
Why is it a pointer?
Can you just do:
AggregationData &Agg = Aggregation[category_str];